### ISSUE TITLE
`run` is extremely slow after upgrade 3.5.1 from 3.5.0 with Window Os

### ISSUE THREAD

**Issue by jongmin4943** [nuxt/nuxt#20990]
### Environment

------------------------------
- Operating System: Windows_NT
- Node Version:     v16.14.0
- Nuxt Version:     3.5.1
- Nitro Version:    2.4.1
- Package Manager:  pnpm@8.3.1
- Builder:          vite
- User Config:      -
- Runtime Modules:  -
- Build Modules:    -
------------------------------

### Reproduction

none

### Describe the bug

`run` is very slow after upgrading to v3.5.1 from v3.5.0.
I tried it with `Window` and `Mac` both and only with `Window` os gets slow after upgrading to nuxt v3.5.1.

![image](https://github.com/nuxt/nuxt/assets/77711322/81a644fe-c2c5-4d91-b3c7-8446da5f1688)
![image](https://github.com/nuxt/nuxt/assets/77711322/96bf8b37-80e4-4927-ba5f-ef36c224a449)

**Comment by danielroe** [comment]
Would you try setting `experimental.watcher` to `chokidar`?

**Comment by ksh9891** [comment]
> Would you try setting `experimental.watcher` to `chokidar`?

![image](https://github.com/nuxt/nuxt/assets/34029686/6a966a64-0a7d-4f83-9285-a22c08440442)

It works.

**Comment by ksh9891** [comment]
> Great - next steps, would you be able to provide a reproduction for the original report? Is this reproducible with a new project?

Sorry, I don't really understand what you are saying `provide a reproduction for the original report`. Do you mean that make a new project with  setting `experimental.watcher` to `chokidar` or set `experimental.watcher` to `chokidar` for original project and see it works?

Just let you know @jongmin4943 is my one of colleages

**Comment by danielroe** [comment]
My apologies for not being clearer. I was asking how the original issue (without the workaround I provided above) can be reproduced.

**Comment by Rasool-deldar** [comment]
I had this problem after upgrading from 3.4.3 to 3.5.0

![image](https://github.com/nuxt/nuxt/assets/58503211/86a56dca-e294-4cf9-890f-aec3a95a0298)

**Comment by AscaL** [comment]
> My apologies for not being clearer. I was asking how the original issue (without the workaround I provided above) can be reproduced.
I'm having the same issue on a completely new repo created with `nuxi init`. I am too on windows
<img width="222" alt="image" src="https://github.com/nuxt/nuxt/assets/8880943/eae71593-e9ec-4380-b22c-cc17c101b7a7">

Cheers

**Comment by grbd** [comment]
Noticed this myself, the quickest way to replicate it is to just setup a docus site
```
npx nuxi init -t themes/docus
```
Then update the package.json to the latest versions, including nuxt 3.5.1

Not sure if it's related
but I'm also noticing the search box under https://content.nuxtjs.org/ is broken as well

**Comment by Sun-ZhenXing** [comment]
I had the same problem with all Nuxt3 projects under Windows. It took too long on **each time** when `dev` (Windows 11 - Intel 12900H):

```
ℹ Vite client warmed up in 4680ms
✔ Nitro built in 349075 ms
✔ Vite server hmr 95 files in 9440.
```

And it recompiles from time to time, on my Windows. It calls msbuild for compilation, which is not expected in minor updates. 

The Node Runtime process consumes over 30% of the CPU and up to 4GB of memory.

There was a small speed increase after setup `experimental: { watcher: 'chokidar' }`, but still spent a lot of time on reloading.

```
ℹ Vite client warmed up in 2355ms
✔ Nitro built in 84662 ms
✔ Vite server hmr 95 files in 2027.28ms
```

**Comment by Sun-ZhenXing** [comment]
> Would you try setting `experimental.watcher` to `chokidar`?

The configuration doesn't work at all in my case.

1. I tried the empty Nuxt project and found that only `3.5.1` had problems.
2. But my own project still has the problem under `3.4.3`
3. Therefore, it should be a problem of some upstream dependency
4. However, there are no exceptions that can be troubleshot through the diff file.

So is there any way to identify the problem?

**Comment by grbd** [comment]
Something also to bear in mind is that I was experencing some strange errors in addition to the above until I tried removing the node_modules and package lock file (pnpm-lock.yaml) to make sure all the dependencies were recalculated from scratch.

If you need more of a working example, this is a site I've been updating recently that ran into the problem with slowness untill I added in that chokidar option
https://github.com/Hecatron-Cad/LinuxCNC.Notes/tree/master/docs/docus

**Comment by ojvribeiro-kore** [comment]
I can confirm the Edge Channel version (3.5.2) is too slow to build the project on Windows.

![image](https://github.com/nuxt/nuxt/assets/126804373/34dcb1e4-5b80-46ed-b6b5-8f33cfcdb6d1)

With the `experimental.watcher: "chokidar"` the build time has significantly improved.

![image](https://github.com/nuxt/nuxt/assets/126804373/ecadc4f9-e155-4242-95ac-046c360d09aa)

**Comment by CharlesBT** [comment]
It doesn't seems to be solved on 3.5.3 same starting time as previous versions on windows

**Comment by Sun-ZhenXing** [comment]
@CharlesBT I don't know if it works for you.

I found that it watch the `node_modules` files in the subproject, so I ignored it and it worked.

So I hope this issue to be fixed.

```ts
export default defineNuxtConfig({
  ignore: [
    '**/*.stories.{js,ts,jsx,tsx}',
    '**/*.{spec,test}.{js,ts,jsx,tsx}',
    '**/*.d.ts',
    '.output',
    '**/-*.*',
    // watcher ignore docs/* to avoid preformance issue
    'docs',
  ],
}
```
